app/lib/hooks/useA11yErrorAnnouncement.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import { AccessibilityInfo } from 'react-native';
+
+interface IUseA11yErrorAnnouncement {
+	errorMessage: string | undefined;
+}
+
+const useA11yErrorAnnouncement = ({ errorMessage }: IUseA11yErrorAnnouncement) => {
+	const [errorAnnounced, setErrorAnnounced] = useState(false);
+
+	useEffect(() => {
+		if (errorMessage && !errorAnnounced) {
+			const message = errorMessage ?? '';
```

Comment:
I don't get the reason for this.
Talked about the `??` vs `||` the other day.
`errorMessage` is tested above, so it's not undefined or falsy.